### PR TITLE
Add symfony phpunit bridge and sulu cms directories to ignore files

### PR DESCRIPTION
**Pull request by alexander-schranz**
When using the symfony phpunit bridge. PHPunit maybe exist in some of this directories. Also added the Testing directory of @sulu as needed from our side.

### CODE REVIEW

File: src/Adapters/Phpunit/Style.php [hunk 1]
```diff
@@ -184,7 +184,10 @@ public function writeError(Throwable $throwable): void
             '/vendor\/mockery\/mockery/',
             '/vendor\/laravel\/dusk/',
             '/vendor\/laravel\/framework\/src\/Illuminate\/Testing/',
-            '/vendor\/laravel\/framework\/src\/Illuminate\/Foundation\/Testing/',
```
Comment (nunomaduro) [inline src/Adapters/Phpunit/Style.php:187]: Why removing this line?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: src/Adapters/Phpunit/Style.php (hunk 1 above)
Comment (alexander-schranz) [reply src/Adapters/Phpunit/Style.php:187]: That was an accident 🙈

Comment (alexander-schranz) [reply src/Adapters/Phpunit/Style.php:187]: Line readded

**Comment by alexander-schranz** [comment]
@nunomaduro Thank you!

**Pull request merged**